secboot/export_test.go

```diff
@@ -0,0 +1,50 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+// +build !nosecboot
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package secboot
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/snapcore/secboot"
+	"github.com/snapcore/snapd/dirs"
+)
+
+func CreateEfivarsSecureBoot(sbData []uint8) {
+	sbFile := filepath.Join(dirs.GlobalRootDir, efivarsSecureBootFile)
+	err := os.MkdirAll(filepath.Dir(sbFile), 0755)
+	if err != nil {
+		panic(err)
```

Comment:
I think returning the error here and using `c.Assert(err,IsNil)` is the more common pattern we have in our codebase